cypress/cypress.config.js

```diff
@@ -67,27 +56,6 @@ module.exports = {
           return global.specRunVersions.get(specRunName) || null;
         },
       });
-      on("before:browser:launch", (browser = {}, launchOptions = {}) => {
-        launchOptions.args = launchOptions.args.filter(
-          (item) => item !== "--disable-dev-shm-usage",
```

Comment:
This gets passed automatically to Chrome as of [Cypress 6](https://docs.cypress.io/guides/references/migration-guide#disable-dev-shm-usage)